websocket-wrappers/holochain/client.js

```diff
@@ -0,0 +1,53 @@
+const path				= require('path');
+const log				= require('@whi/stdlog')(path.basename( __filename ), {
+    level: process.env.LOG_LEVEL || 'fatal',
+});
+
+const { AdminWebsocket, AppWebsocket }				= require('@holochain/conductor-api');
+const ConnectionMonitor           = require('../utils.js');
+
+const HOLOCHAIN_WS_CLIENT_OPTS			= {
+    "reconnect_interval": 1000,
+    "max_reconnects": 300,
+};
+
+class HcAdminWebSocket extends AdminWebsocket {
```

Comment:
We should not have this wrapper here. This is just complicating the code. 

If we need a ConnectionMonitor we should add it to the `condcutor-api`, that is where it should actually live.

So that we can use it easily use it here and any other repo.

It looks like the previous implementation needed it because it was a wrapper around `rpc-websockets` that we did not have control over. But we do have access to the AdminWebsocket now.

Reply:
I agree that this would ideally live in `holochain-conductor-api` repo, but think that this is a bit of scope creep.  I'll make an issue for this and bring it to Rob to track on the board.